lib/helper/request-store.js

```diff
@@ -60,13 +60,33 @@ const putArtifact = async (buildId, token, fileName, file) => {
     };
 
     try {
-        return request(options);
+        return await request(options);
     } catch (err) {
         throw new Error(err.response.body.toString());
     }
 };
 
+const deleteZipArtifact = async (buildId, token) => {
+    const options = {
+        url: `${baseUrl}/builds/${buildId}/ARTIFACTS/${ZIP_FILE}`,
+        method: 'DELETE',
+        headers: {
+            Authorization: token
+        },
+        retry: {
+            limit: RETRY_LIMIT
+        }
+    };
+
+    try {
+        return await request(options);
+    } catch (err) {
+        throw new Error(err.response.body);
```

Comment:
Do we need `toString()` same as other functions?
```suggestion
        throw new Error(err.response.body.toString());
```

Reply:
thx.